hoot-core/src/main/cpp/hoot/core/conflate/ReviewMarker.cpp

```diff
@@ -168,6 +168,36 @@ void ReviewMarker::mark(const OsmMapPtr &map, const ElementPtr& e1, const Elemen
   map->addElement(r);
 }
 
+void ReviewMarker::mark(const OsmMapPtr &map, set<ElementId> ids, const QString& note,
+   const QString& reviewType, double score, vector<QString> choices)
+{
+  if (note.isEmpty())
+  {
+    throw IllegalArgumentException("You must specify a review note.");
+  }
+
+  RelationPtr r(new Relation(Status::Conflated, map->createNextRelationId(), 0, Relation::REVIEW));
+  r->getTags().set(_reviewNeedsKey, true);
+  r->getTags().appendValueIfUnique(_reviewNoteKey, note);
+  r->getTags().appendValueIfUnique(_reviewTypeKey, reviewType);
```

Comment:
@mingjiesu,

Because the tags list is empty on a new relation you can just use `set` rather than `appendValueIfUnique`.